helm/idol-qms/values.yaml

```diff
@@ -0,0 +1,126 @@
+
+# existingConfigMap -- if specified, mounted at /etc/config/idol and expected to provide community.cfg
+existingConfigMap: ""
+
+# -- Additional PodSpec Volume (see https://kubernetes.io/docs/reference/kubernetes-api/workload-resources/pod-v1/#volumes)
+additionalVolumes: []
+
+# -- Additional PodSpec VolumeMount (see https://kubernetes.io/docs/reference/kubernetes-api/workload-resources/pod-v1/#volumes-1)
+additionalVolumeMounts: []
+
+# -- Default configuration for [PromotionAgentStore]::Host
+agentStoreName: &agentStoreName "qms-agentstore"
+# -- Default configuration for [PromotionAgentStore]::Port
+agentStoreACIPort: &agentstoreACIPort "9150"
+
+# -- Default configuration for [IDOL]::Host
+queryserviceName: &queryServiceName "idol-content"
+# -- Default configuration for [IDOL]::Port
+queryserviceACIPort: &queryServiceACIPort "9100"
+
+##### INGRESS VARIABLES #####
+
+ingress:
+  # -- Create ingress resource
+  enabled: true
+
+  # -- Ingress controller type to setup for.
+  # Valid values are nginx or haproxy (used by OpenShift)
+  type: nginx
+
+  # -- Optional parameter to override the default ingress class
+  className: ""
+
+  # -- Optional host (see https://kubernetes.io/docs/concepts/services-networking/ingress/#ingress-rules).
+  # For an OpenShift environment this is required (see https://docs.openshift.com/container-platform/4.11/networking/routes/route-configuration.html#nw-ingress-creating-a-route-via-an-ingress_route-configuration)
+  host: ""
+
+##### INTERNAL VARIABLES #####
+# name -- used to name deployment, service, ingress
+name: "idol-qms"
+# aciPort -- port service will serve ACI connections on
+aciPort: "16000"
+# servicePort -- port service will serve service connections on
+servicePort: "16002"
+# livenessProbe -- container livenessProbe settings
+livenessProbe:
+  initialDelaySeconds: 120
+
+singleAgentstore:
+
+  # -- whether to deploy the single-content sub-chart that uses an IDOL Agentstore
+  #    configuration file and docker image.
+  enabled: true
+
+  # -- the config map to use for providing a qms-agentstore configuration.
+  existingConfigMap: "qms-agentstore-cfg"
+
+  idolImage:
+    # -- overrides the default value for single-content.idolImage.repo ("content")
+    #    to guarantee that we use an IDOL Agentstore docker image.
+    repo: "qms-agentstore"
+
+  # -- agentstore port service will serve ACI connections on
+  aciPort: *agentstoreACIPort
+
+  ingress:
+    # -- the ingress controller path to access the agentstore query service with
+    path: "/agentstore/"
+    # -- the ingress controller path to access the agentstore index service with
+    indexPath: "/agentstore-index/"
+
+  # -- the agentstore engine's query service ACI port
+  queryserviceACIPort: *agentstoreACIPort
+
+  # -- the agentstore engine's index service name
+  indexserviceName: "idol-agentstore-index-service"
+
+  # -- used to name deployment, service, ingress
+  name: *agentStoreName
+
+single-content:
```

Comment:
should add sub-key to stop us from attempting to deploy duplicate idol-licenseserver subcharts.
```
  idol-licenseserver.enabled: false
```

Reply:
https://github.com/opentext-idol/idol-containers-toolkit/pull/27/commits/5736e0a5828c86ace50a9cffe68fea3b890af269